valohai_yaml/objs/input.py

```diff
@@ -37,13 +37,15 @@ def __init__(
         description: Optional[str] = None,
         keep_directories: KeepDirectoriesValue = False,
         filename: Optional[str] = None,
+        override_mode: Optional[bool] = None,
```

Comment:
I don't think this type annotation is correct. The default should be the current legacy default.